internal/local/structure_v2.go

```diff
@@ -534,6 +569,64 @@ func writeSync(rootDir string, sync SyncStructure) error {
 	)
 }
 
+func writeSchemas(rootDir string, schemas []map[string]interface{}) error {
+	dir := filepath.Join(rootDir, NameDataSources)
+	if err := os.MkdirAll(dir, os.ModePerm); err != nil {
+		return err
+	}
+	for _, schema := range schemas {
+		schemaBody := schema[NameSchema]
+		if schemaBody == nil {
+			schemaBody = map[string]interface{}{}
+		}
+		dataSchema, err := MarshalJSON(schemaBody)
+		if err != nil {
+			return err
+		}
+		relationships := schema[NameRelationships]
+		if relationships == nil {
+			relationships = map[string]interface{}{}
+		}
+		dataRelationships, err := MarshalJSON(relationships)
+		if err != nil {
+			return err
+		}
+
+		metadata, ok := schema["metadata"].(map[string]interface{})
+		if !ok {
+			return errors.New("error writing schemas")
+		}
+		if metadata == nil {
+			metadata = map[string]interface{}{}
+		}
+
+		dataSource, _ := metadata["data_source"].(string)
```

Comment:
[nit] are there constants we could use for `data_source`, `database`, and `collection`?